server/repository/src/mock/test_requisition_service.rs

```diff
@@ -159,7 +162,7 @@ pub fn mock_draft_response_requisition_for_update_test() -> RequisitionRow {
         r.name_id = "name_a".to_owned();
         r.store_id = mock_store_a().id;
         r.r#type = RequisitionRowType::Response;
-        r.status = RequisitionRowStatus::Draft;
+        r.status = RequisitionRowStatus::New;
```

Comment:
Please rename the fn name from `mock_draft_response_requisition_for_update_test` to `mock_new_response_requisition_for_update_test` + rename id string...